content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -0,0 +1,432 @@
+    
+
+### Persistent Volume last phase transition time
+
+Persistent Volume last phase transition time feature moved to GA in v1.31. 
+This feature adds a `PersistentVolumeStatus` field which holds a timestamp of when a PersistentVolume last transitioned to a different phase.
+With this feature enabled, every PersistentVolume object will have a new field `.status.lastTransitionTime`, that holds a timestamp of
+when the volume last transitioned its phase. 
+This change is not immediate; the new field will be populated whenever a PersistentVolume is updated and first transitions between phases (`Pending`, `Bound`, or `Released`) after upgrading to Kubernetes v1.31.
+This allows you to measure time between when a PersistentVolume moves from `Pending` to `Bound`. This can be also useful for providing metrics and SLOs.
+
+For more details about this feature please visit the [PersistentVolume documentation page](/docs/concepts/storage/persistent-volumes/).
+
+This work was done as a part of [KEP #3762](https://github.com/kubernetes/enhancements/issues/3762) by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+## Highlights of features graduating to Beta
+
+_This is a selection of some of the improvements that are now beta following the v1.31 release._
+    
+### nftables backend for kube-proxy
+
+The Kubernetes project intends to update kube-proxy on Linux so that it uses `nftables`,
+not `iptables`, as the default implementation. The nftables backend moves to beta in v1.31,
+behind a newly added feature gate, `NFTablesProxyMode` (disabled by default).
+In this mode, kube-proxy configures packet forwarding rules using the nftables API of the kernel netfilter subsystem. 
+For each endpoint, it installs nftables rules which, by default, select a backend Pod at random.
+
+The nftables API is the successor to the iptables API and is designed to provide better performance and scalability than iptables. 
+The `nftables` proxy mode is able to process changes to service endpoints faster and more efficiently than the `iptables` mode, and is also able to more efficiently process packets in the kernel (though this only
+becomes noticeable in clusters with tens of thousands of services).
+
+As of Kubernetes v1.31, the `nftables` mode is still relatively new, and may not be compatible with all network plugins; consult the documentation for your network plugin. 
+This proxy mode is only available on Linux nodes, and requires kernel 5.13 or later.
+See the full [migration guide](/docs/reference/networking/virtual-ips/#proxy-mode-nftables) for more details. 
+
+This work was done as part of [KEP #3866](https://github.com/kubernetes/enhancements/issues/3866) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+### Changes to reclaim policy for PersistentVolumes
```

Comment:
@xing-yang can you review the Changes to reclaim policy for PersistentVolumes section?

Reply:
@deepakkinni can you review this section if you have a minute?

Reply:
looks fine to me.

Reply:
Looks good to me as well.